proto/types.proto

```diff
@@ -36,3 +36,42 @@ message Guid {
   // 128 bits (16 bytes)
   bytes value = 1;
 }
+
+enum HaRole{
+  HA_SCOPE_ROLE_DEAD = 0;
```

Comment:
Please add `HA_SCOPE_ROLE_UNSPECIFIED` as the first item/index 0

Reply:
done.